### DISCUSSION TITLE
Properties of audio files should show duration

### DISCUSSION THREAD

**Discussion by RustoMCSpit** [linuxmint/discussions#281] [Ideas]
And if multiple files are selected, combined and average length too

**Comment by claudiux** [comment]
`apt install nemo-media-columns`
Then, you can add a column "duration" in nemo. But no combined and average length.

**Reply by RustoMCSpit** [reply]
no this is a workaround, the above should be default. i cant think of a logical use case wherr including that info wouldnt be useful

**Comment by mtwebster** [comment]
This isn't something for core nemo, but [nemo-audio-tab](https://github.com/linuxmint/nemo-extensions/tree/master/nemo-audio-tab) could be expanded to allow showing properties for multiple selections at once.

There are also literally dozens of music player/library/media-hub-type programs out there as well, that would be better at this.

**Reply by RustoMCSpit** [reply]
The duration of an audio clip isn't worth displaying in properties by default?

**Reply by mtwebster** [reply]
No, it's not. Duration is not something available in standard file properties, it needs an external library or program to accomplish this, which is one of the reasons we have extensions.

**Reply by RustoMCSpit** [reply]
>standard file properties

how do we modify that
